packages/gguf/src/gguf.ts

```diff
@@ -0,0 +1,325 @@
+type MetadataBaseValue = string | number | bigint | boolean;
+type MetadataValue = MetadataBaseValue | MetadataBaseValue[] | MetadataValue[]; /// recursive as arrays can be nested.
```

Comment:
```suggestion
export type MetadataBaseValue = string | number | bigint | boolean;
export type MetadataValue = MetadataBaseValue | MetadataBaseValue[] | MetadataValue[]; /// recursive as arrays can be nested.
```

I'd need them to be exported for strongly typing the frontend metadata table

Reply:
sounds good, i let you do your final tweaks and merge whenever you want ok?

Reply:
yep